tests/moduletemplates/moduletemplate_template_operator_v2_direct_version.yaml

```diff
@@ -0,0 +1,78 @@
+apiVersion: operator.kyma-project.io/v1beta2
+kind: ModuleTemplate
+metadata:
+  name: template-operator-2.4.2-e2e-test
+  namespace: kcp-system
+  annotations:
+    "operator.kyma-project.io/doc-url": "https://kyma-project.io"
+    "operator.kyma-project.io/is-cluster-scoped": "false"
+spec:
+  channel: none
+  mandatory: false
+  moduleName: template-operator
+  version: 2.4.2-e2e-test
+  data:
+    apiVersion: operator.kyma-project.io/v1alpha1
+    kind: Sample
+    metadata:
+      name: sample-2.4.2-e2e-test
+    spec:
+      resourceFilePath: "./module-data/yaml"
+  descriptor:
```

Comment:
The descriptor here must be copied to be in a new format

Reply:
it's not necessary that fixed version must use new OCM format, both new and old format should compatible with fixed version moduletemplate.